README.rst

```diff
@@ -245,7 +245,7 @@ Let us test the client-to-server connection:
 .. code-block:: console
 
     $ reana-client ping
-    Server is running.
+    Connected to https://reana.cern.ch - Server is running.
```

Comment:
I have just realised that we are missing setting the `REANA_ACCESS_TOKEN` env var (I guess around L233 here).  Also, in L241 we should do `reana-cluster env --all` instead (related to https://github.com/reanahub/reana-cluster/pull/108). I have already merged https://github.com/reanahub/reana-demo-root6-roofit/pull/23 we need to add it there as well.